### ISSUE TITLE
用append添加一行，在删除一行(removeByUniqueId)，之后再添加一行。在调用getSelections方法之后，取不到所选的行

### ISSUE THREAD

**Issue by lianglinux5** [wenzhixin/bootstrap-table#2900]
```
--- a/his_biz_main/src/main/webapp/js/bootstraptable/bootstrap-table.js
+++ b/his_biz_main/src/main/webapp/js/bootstraptable/bootstrap-table.js
@@ -1849,7 +1849,7 @@

             var $this = $(this),
                 checked = $this.prop('checked'),
-                row = that.data[$this.data('index')];
+                row = that.options.data[$this.data('index')];

             if (that.options.maintainSelected && $(this).is(':radio')) {
                 $.each(that.options.data, function (i, row) {
```

**Comment by djhvscf** [comment]
Can you translate this issue to english, please?

**Comment by wenzhixin** [comment]
ref #2831

**Comment by Falseee** [comment]
@wenzhixin is this issue fixed by #2831 ?

**Comment by wenzhixin** [comment]
Yes
